.golangci.yml

```diff
@@ -5,6 +5,42 @@ run:
   deadline: 3m
   tests: true
 
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+  skip-dirs:
+    - vendor
+
+# output configuration options
+output:
+  # colored-line-number|line-number|json|tab|checkstyle, default is "colored-line-number"
+  format: colored-line-number
+
+  # print lines of code with issue, default is true
+  print-issued-lines: true
+
+  # print linter name in the end of issue text, default is true
+  print-linter-name: true
+
+linters:
+  enable:
+    # Sorted alphabetically.
+    - errcheck
+    - gochecknoglobals
```

Comment:
I would not even mind also removing `gochecknoglobals` since rely on globals in some parts of code and we're fine living with it, but that's optionally.

Reply:
Ack, let me remove! 
Thanks for your hint about downstream, I did have that in mind! 🙂